suricata.yaml.in

```diff
@@ -1700,8 +1700,13 @@ detect:
     toserver-groups: 25
   sgh-mpm-context: auto
   inspection-recursion-limit: 3000
-  # maximum number of times a tx will get logged for a stream-only rule match
+  # maximum number of times a tx will get logged for rules without app-layer keywords
   # stream-tx-log-limit: 4
+  # try to tie an app-layer transaction for rules without app-layer keywords
+  # if there is only one live transaction for the flow
+  # allows to log app-layer metadata in alert
+  # but the transaction may not be the relevant one.
+  # guess-applayer-tx: no
```

Comment:
Is `stream-tx-log-limit` referring to app-layer tx?  Super picky, but if so, we already have a naming scheme that `guess-applayer-tx` does not follow.  Should it just be `guess-tx`?

Reply:
> Is stream-tx-log-limit referring to app-layer tx

Yes, but `stream-tx-log-limit` seems a bad name, and a bad example to follow

Reply:
What should we do in such a case?